systemd/kibana.service

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=Kibana 4
+BindsTo=probemanager.service
+Wants=elasticsearch.service
+After=elasticsearch.service
```

Comment:
Is all this logic needed. Can't it be simplified and only keep the 
```
Description=Kibana 4
BindsTo=probemanager.service
Wants=elasticsearch.service
After=elasticsearch.service
```

Potentially with this also:
```
Wants=probemanager.service
Before=probemanager.service
```